src/wp-includes/link-template.php

```diff
@@ -4539,9 +4539,11 @@ function get_avatar_data( $id_or_email, $args = null ) {
 function get_theme_file_uri( $file = '' ) {
 	$file = ltrim( $file, '/' );
 
+	$stylesheet_directory = get_stylesheet_directory();
+
 	if ( empty( $file ) ) {
 		$url = get_stylesheet_directory_uri();
-	} elseif ( file_exists( get_stylesheet_directory() . '/' . $file ) ) {
+	} elseif ( $stylesheet_directory !== get_template_directory() && file_exists( $stylesheet_directory . '/' . $file ) ) {
```

Comment:
In case you have not yet noticed, you created a `$template_directory` variable for `get_theme_file_path()`, and you may want to do the same in this function too.

Reply:
This is by design. `get_template_directory()` is only used one in this function.